pyproject.toml

```diff
@@ -129,12 +129,13 @@ target-version = "py310"
 
 [tool.ruff.lint.per-file-ignores]
 "__init__.py" = ["F401"]
-"tests/*.py" = ["S101"]
-"examples/*.py" = ["T201"]
+"tests/*.py" = ["S101", "ANN"]
+"examples/*.py" = ["T201", "ANN401"]
```

Comment:
hmm, do we want to allow for `Any` in examples?

Reply:
ignore `'ANN` in examples